packages/build/src/plugins_core/secrets_scanning/utils.ts

```diff
@@ -391,6 +407,143 @@ const searchStream = (basePath: string, file: string, keyValues: Record<string,
   })
 }
 
+/**
+ * Search stream implementation using just read stream that allows to buffer less content
+ */
+const searchStreamNoReadline = (
```

Comment:
It would be very lovely to have `searchStreamNoReadline` marked up with guiding comments similar to what we had in the original implementation